sdk/storage/azure-storage-blob-cryptography/pom.xml

```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-security-keyvault-keys</artifactId>
-      <version>4.3.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-security-keyvault-keys;dependency} -->
+      <version>4.2.1</version> <!-- {x-version-update;unreleased_com.azure:azure-security-keyvault-keys;dependency} -->
```

Comment:
@vcolin7 Could you confirm if we are releasing key beta with `4.3.0-beta.1` if yes, then we should not change this.

Reply:
This is just for the from source pipeline to work. This change won't go into master.